src/Illuminate/Redis/Connections/PacksPhpRedisValues.php

```diff
@@ -82,6 +82,17 @@ public function pack(array $values): array
         return array_map($processor, $values);
     }
 
+    /**
+     * Determine if JSON serialization is enabled.
+     *
+     * @return bool
+     */
+    public function jsonSerialized(): bool
```

Comment:
Should this be something like `usesJsonSerialization`?

Reply:
Yes, name fits better. Will rename it

Reply:
Renamed it to `usesJsonSerialization`.